app/models/competition.rb

```diff
@@ -95,6 +95,8 @@ class Competition < ApplicationRecord
     restricted: 2,
   }, prefix: true
 
+  enum :user_can_cancel, [:always, :not_accepted, :unpaid], prefix: :can_cancel
```

Comment:
If you just pass `prefix: true`, then the name of the enum will become the prefix. Arguably cleaner and there's not much of a difference between `user_can_cancel_always` and `can_cancel_always`.

Reply:
Also, maybe `competitor` instead of `user`?

Reply:
I like `competitor` instead of `user`, and agree that specifying a different prefix is unnecessary with the new name